### PR TITLE
v5: Add revision history info, drop NEW markers

### PR DESCRIPTION
**Pull request by raffenet**
This PR contains 2 commits.
1. The review history section for v5 as presented at the Q3 2022 ASC meeting.
2. Drops all "NEW" markers for declarations in the document. As discussed at the same Q3 2022 meeting, these markers are error-prone, and somewhat obsolete with the addition of the revision history section. Consensus was to remove them from the document.

### CODE REVIEW

File: Chap_Revisions.tex [hunk 1]
```diff
@@ -1433,15 +1433,37 @@ \subsection{Added Attributes (Provisional)}
...
+  \item Add specific values to constant definitions to ensure
+    consistency across implementations.
+  \item Add Use-Cases appendix with descriptions for Business Card
+    Exchange, Debugging, Hybrid Applications, MPI Sessions, and
+    Cross-Version Compatibility.
+  \item Add guidance on how PMIx defines an Application Binary Interface (ABI).
+  \item Add ABI query attributes.
+  \item Add example C and Python PMIx client codes in sources/ directory.
+  \item Clarify three roles of consumers of the PMIx interface (client, server, tool).
+  \item Clarify when PMIX_PARENT_ID attribute is provided.
```
Comment (jjhursey) [inline Chap_Revisions.tex:1448]:
```suggestion
  \item Clarify when \refattr{PMIX_PARENT_ID} attribute is provided.
```

**Review by jjhursey — CHANGES_REQUESTED** [review]
Looks good. A few minor edits for the revision log

File: Chap_Revisions.tex [hunk 2]
```diff
@@ -1433,15 +1433,37 @@ \subsection{Added Attributes (Provisional)}
...
+  \item First release prepared using procedures defined in the PMIx
+    Governance document.
+  \item Add specific values to constant definitions to ensure
+    consistency across implementations.
+  \item Add Use-Cases appendix with descriptions for Business Card
+    Exchange, Debugging, Hybrid Applications, MPI Sessions, and
+    Cross-Version Compatibility.
+  \item Add guidance on how PMIx defines an Application Binary Interface (ABI).
+  \item Add ABI query attributes.
+  \item Add example C and Python PMIx client codes in sources/ directory.
```
[the remark this answers is not in the document]
Comment (raffenet) [reply Chap_Revisions.tex:1446]: That's fine with me. I was just going thru the PRs and went back and forth on whether this one belonged.

**Review by jjhursey — APPROVED** [review]

**Comment by raffenet** [comment]
@jjhursey @dsolt I added 3 more minor commits to this branch since presenting at the quarterly. Please take a look.
1. simple typo fix
2. add reference to new Use-Cases appendix
3. add item about removing NEW from declarations

**Review by jjhursey — DISMISSED** [review]

**Comment by jjhursey** [comment]
This LGTM. I think if @dsolt is good with it then we can merge whenever you are ready.

**Review by dsolt — DISMISSED** [review]

**Pull request merged**